app/controllers/admin/users_controller.rb

```diff
@@ -19,16 +19,16 @@ def show
 	end
 
 	def update
-		if @user.update_attributes(secure_params)
-			redirect_to users_path, :notice => "User updated."
+		if @user.update(secure_params)
+			redirect_to admin_user_path(@user), :flash_success_info => "User role was updated."
```

Comment:
```suggestion
			redirect_to admin_user_path(@user), flash_success_info: 'User role was updated.'
```

@BALAJI-BALA - use new ruby syntax and single quotes. Change in all the other files.